bouture.js

```diff
@@ -63,6 +64,17 @@ function completeElement (tag, args) {
               return name.replace(/^once|on/, '').toLowerCase()
             }
 
+            function isAttribute (name) {
+              if (name.match(/^data/)) {
+                return name.match(/^data-([a-z]|[0-9]|-)*$/)
```

Comment:
Are you sure about `([a-z]|[0-9]|-)*` ?

Reply:
Yes, it's wrong.
I propose a solution based on this article: https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dataset